mozilla-release/browser/installer/allowed-dupes.mn

```diff
@@ -226,3 +226,35 @@ res/table-remove-row.gif
 browser/chrome/browser/content/browser/defaultthemes/devedition.icon.png
 browser/chrome/browser/content/branding/icon64.png
 browser/chrome/devtools/content/framework/dev-edition-promo/dev-edition-logo.png
+# CLIQZ
+browser/chrome/browser/skin/classic/browser/downloads/download-glow-menuPanel-XPVista7.png
+browser/chrome/browser/skin/classic/browser/downloads/download-glow-menuPanel.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/urlbar-arrow.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/urlbar-arrow.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/urlbar-arrow.png
+browser/chrome/browser/skin/classic/browser/cliqz/urlbar-arrow.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/urlbar-arrow@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/urlbar-arrow@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/urlbar-arrow@2x.png
+browser/chrome/browser/skin/classic/browser/cliqz/urlbar-arrow@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/close.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/close.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/close.png
+browser/chrome/browser/skin/classic/browser/cliqz/close.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/close@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/close@2x.png
+browser/chrome/browser/skin/classic/browser/cliqz/close@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/Toolbar.png
+browser/chrome/browser/skin/classic/browser/Toolbar.png
+browser/chrome/browser/skin/classic/browser/Toolbar-aero.png
+browser/chrome/browser/skin/classic/browser/Toolbar-lunaSilver.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/win/Toolbar@2x.png
+browser/chrome/browser/skin/classic/browser/Toolbar@2x.png
+browser/features/cliqz@cliqz.com/icon.png
+browser/features/cliqz@cliqz.com/chrome/content/static/skin/cliqz.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/menuPanel.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/menuPanel.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/Toolbar@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/Toolbar@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/linux/menuPanel@2x.png
+browser/features/cliqz@cliqz.com/chrome/content/theme/images/browser/mac/menuPanel@2x.png
```

Comment:
It always was there, but in FF52 Mozilla's start more aggressively work with duplicates. Instead of warning (before 52) they putting an Error now. I will fix this in another way - will use --warning flag for find-dupes.py script